src/hotspot/share/oops/instanceKlass.hpp

```diff
@@ -1125,7 +1125,7 @@ class InstanceKlass: public Klass {
   void compute_has_loops_flag_for_methods();
 #endif
 
-  u2 compute_modifier_flags() const;
+  int compute_modifier_flags() const;
```

Comment:
I don't see why this can't stay u2.

Reply:
I had some compilation error for conversion that has disappeared into the either with u2, so I've restored them to u2.